Automattic/woocommerce-payments
# Fix total price for Google Pay on quantity change.

Fixes #6842 

#### Changes proposed in this Pull Request
Fix `get_selected_product_data`. Total price should be the price * quantity. At this point the response reflects the price of qty 1.








#### Testing instructions




Using develop branch:
* Navigate to a single product where Google Pay displays
* Increase the quantity
* Click "Buy with GPay"
* Result matches expecations
* Exit the modal
* Change the quantity to something different and above 1
* Click "Buy with Gpay" again
* The price is now that of a single unit
* checkout this branch
* follow the steps above. The total price should take into account also the product quantity.

-------------------


**Post merge**
